VideoLocker/res/layout/fragment_course_unit_webview.xml

```diff
@@ -15,4 +15,10 @@
         android:layout_gravity="center_vertical|center_horizontal"
         android:visibility="gone" />
 
+    <TextView
+        android:id="@+id/content_unavailable_error_text"
+        style="@style/content_unavailable_error_text"
+        android:text="@string/reset_no_network_message"
+        android:visibility="gone" />
```

Comment:
`android:visibility="gone"` is already defined in the `content_unavailable_error_text ` style, and can be removed.